src/dashboards.js

```diff
@@ -13,13 +13,16 @@ $(document).ready(function() {
       maxHeight: 500,
       disableIfEmpty: true,
       nSelectedText: $this.data("n-selected") !== undefined ? $this.data("n-selected") : "selected",
+      onDropdownShow: function(event) { // Focus the search box whenever the dropdown is opened
+        setTimeout(function() { $(event.currentTarget).find(".filter input").focus(); }, 0);
+      },
     };
     if ($this.attr("id") === "measure") {
       options.filterBehavior = "custom";
       options.filterCallback = function(element, query) {
         var value = $(element).find("label").text().toLowerCase();
```

Comment:
value is not a good variable name here, is this the search string or one of the options in the dropdown list?